stdlib/public/core/UTF8SpanBits.swift

```diff
@@ -117,6 +124,9 @@ extension UTF8Span {
     0xFF00_0000_0000_0000
   }
 
+  /// The number of UTF-8 code units stored
```

Comment:
```suggestion
  /// The number of UTF-8 code units stored.
```

Is the word "stored" needed here?  If so, can we specify _where_ they're stored?

Reply:
That sounds great!